.dev/docker/env/Dockerfile

```diff
@@ -36,44 +34,55 @@ RUN apt-get update \
       libusb-1.0-0-dev \
       libvtk${VTK_VERSION}-dev \
       libvtk${VTK_VERSION}-qt-dev \
+      lsb-release \
       qtbase5-dev \
       software-properties-common \
+      wget \
+      xvfb \
+ && if [ "$USE_LATEST_CMAKE" = true ] ; then \
+      wget -O - https://apt.kitware.com/keys/kitware-archive-latest.asc 2>/dev/null | gpg --dearmor - | tee /etc/apt/trusted.gpg.d/kitware.gpg >/dev/null ; \
```

Comment:
Any reason to use tee instead of writing into the file directly?

Reply:
Don't know, it is copy & paste from their documentation. Maybe because of `gpg --dearmor`? But the documentation of `dearmor` doesn't sound like it is really needed:

> Pack or unpack an arbitrary input into/from an OpenPGP ASCII armor. This is a GnuPG extension to OpenPGP and in general not very useful.

Reply:
We could try to throw away tee in that case

Reply:
Used `apt-key add -` now instead like it is used by [llvm.sh](https://apt.llvm.org), even it is [not fully recommended](https://unix.stackexchange.com/a/582853).

Reply:
Maybe I well switch back to `gpg` & `tee` as `apt-get` [seems to be deprecated](https://www.linuxuprising.com/2021/01/apt-key-is-deprecated-how-to-add.html)

Reply:
Adjusted:

```sh
apt-add-repository "deb [signed-by=/usr/share/keyrings/kitware-keyring.gpg] https://apt.kitware.com/ubuntu/ $cmake_ubuntu_version main"
```
gave me always `Error: 'deb [signed-by=/usr/share/keyrings/kitware-keyring.gpg] https://apt.kitware.com/ubuntu/ bionic main' invalid`

So I switched to
```sh
echo "deb [signed-by=/usr/share/keyrings/kitware-keyring.gpg] https://apt.kitware.com/ubuntu/ $cmake_ubuntu_version main" > /etc/apt/sources.list.d/kitware.list
apt-get update ;
```